src/styles/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &__star {
+    content: '';
+    width: 16px;
+    height: 16px;
+
+    @include no-active-stars;
+
+    &:not(&:last-child) {
+      margin-right: $commonMargin;
+    }
+  }
+
+  &--active &__star:nth-child(-n + #{$activeStarsAmount}) {
```

Comment:
Consider using styles from the previous task to keep modifiers working

Reply:
Thanks, done!